seq2science/workflows/scrna_seq/config.yaml

```diff
@@ -18,7 +16,13 @@ technical_replicates: merge
 # seq2science does currently not support scrna-seq platforms that generate more than two fastq files, such as 10xv1.
 quantifier:
   kallistobus:
-    ref: '--lamanno'
-    count: '-x 0,8,16:0,0,8:1,0,0 --verbose --lamanno'
-    #count: '-x 10XV2 --verbose --lamanno'
+    ## Quantification example ##
+    # count: '-x 0,8,16:0,0,8:1,0,0 --h5ad --verbose'
+    # count: '-x 10XV3 --h5ad --verbose'
+
+    ## Velocity example ##
+    # ref: '--workflow lamanno'
+    # count: '-x 0,8,16:0,0,8:1,0,0 --h5ad --verbose --workflow lamanno'
+    # count: '-x 10XV3 --verbose --h5ad --workflow lamanno'
+
```

Comment:
What happens if you leave both uncommented?

Reply:
Then it will probably fail. I don't think its possible to set sane defaults for Kallisto bus since it highly depends on the technology you use and workflow (quantification, velocity). For now, it performs 10xv3 scRNA quantification by default.